### PR TITLE
chore: happy clippy

### PR DESCRIPTION
**Pull request by dignifiedquire**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -73,7 +73,7 @@ jobs:
       - *restore-cache
       - run:
           name: Run cargo clippy
-          command: cargo clippy --all-targets --all-features --workspace -- -D warnings
+          command: cargo clippy --all-features --workspace -- -D warnings
```
Comment (vmx) [inline .circleci/config.yml:76]: Was that change intentional? I think checking tests and examples is a good idea.

Comment (vmx) [reply .circleci/config.yml:76]: If running a nightly Clippy isn't a problem, the fix is `cargo +<< pipeline.parameters.nightly-version >> clippy --all-targets --all-features --workspace -- -D warnings` (and removing the `--profile minimal from the nightly installation`.

Comment (dignifiedquire) [reply .circleci/config.yml:76]: that is probably a better solution

**Review by vmx — APPROVED** [review]
@dignifiedquire I guess this one is now ready to go.

**Comment by vmx** [comment]
@dignifiedquire Was this an intentional close without merging? If yes, why?

**Comment by dignifiedquire** [comment]
> @dignifiedquire Was this an intentional close without merging? If yes, why?

I was confused
